pkg/prow/config/tide.go

```diff
@@ -0,0 +1,133 @@
+package config
+
+import (
+	"fmt"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/jenkins-x/jx/pkg/util"
+	"k8s.io/test-infra/prow/config"
+	"time"
+)
+
+// CreateTide creates a default Tide Config object
+func CreateTide() config.Tide {
+	// todo get the real URL, though we need to handle the multi cluster use case where dev namespace may be another cluster, so pass it in as an arg?
+	t := config.Tide{
+		TargetURL: "https://tide.foo.bar",
```

Comment:
I would inject the `TargetURL` as an argument in the `CreateTide` function instead of hardcoding a dummy value here

Reply:
not my code. No idea why this is even here.

Reply:
I just moved the methods to a new package